src/reference/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -86,6 +86,10 @@ The xref:jdbc/dsl.adoc[] chapter provides more details.
 The `JdbcLock` now supports the feature of customized time-to-live for the lock status data.
 See xref:jdbc/lock-registry.adoc[] for more information.
 
+The `JdbcChannelMessageStore` now supports built-in Jackson JSON serialization as an alternative to Java serialization, enabling human-readable message storage.
+New components `JacksonChannelMessageStorePreparedStatementSetter` and `JacksonMessageRowMapper` provide this functionality with JSON-specific database schemas for PostgreSQL, MySQL, and H2.
```

Comment:
I don't think it has any specific value to mention those DBs. 
I believe that any CLOB-supporting DB is OK to have a JSON as its value.
So, your statement about `JSON-specific database schemas` does not reflect reality somehow...
Maybe you meant those custom schema files?
Then, OK: since we talk about dropping them in favor of docs, then this sentence would be changed as well.